### PR TITLE
docs: Add FAQ about incorrect VPC settings

### PR DESCRIPTION
**Pull request by kayousterhout**
This commit adds a new FAQ section to the docs, and adds one
question to that section about how to fix broken default VPC
settings on Amazon.

### CODE REVIEW

File: docs/_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Frequently Asked Questions
+
+This section includes answers to common questions about Quilt, and solutions
+to various issues.  If you run into an issue and can't find the answer here,
+don't hesitate to email us at [discuss@quilt.io](mailto:discuss@quilt.io).
+
+### I tried to `quilt run` a blueprint on Amazon and nothing seems to be working.
```
Comment (kklin) [inline docs/_FAQ.md]: It's worth mentioning that the machines will show as connected, and containers will show up in `quilt ps`, but they'll never show up in the `scheduled` state. (the status will be empty, or `building` for deployed Dockerfiles)

Comment (kayousterhout) [inline docs/_FAQ.md]: Thanks for noting this -- fixed!

File: docs/_FAQ.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+If you're running a blueprint on AWS and the containers are not getting properly
+created, you may have an issue with your VPC (Virtual Private Cloud) settings
+on Amazon.  When this issue occurs, if you run `quilt ps`, the machines will all
```
Comment (kklin) [inline docs/_FAQ.md]: Should change `qulit ps` to `quilt show`

File: docs/_FAQ.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+have status `connected`, but the containers will never progress to the
+`scheduled` state (either the status will be empty, or for Dockerfiles that are
+built in the cluster, the status will say `building`).  This issue only occurs
```
Comment (kklin) [inline docs/_FAQ.md]: Sorry, I meant `built` rather than `building`.

File: docs/_FAQ.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+if you've changed your default VPC on Amazon, so if you don't know what a VPC is
+or you haven't used one before on Amazon, this is probably not the issue you're
+experiencing.
+
+If you previously changed your default VPC on Amazon, it may be configured in a
```
Comment (kklin) [inline docs/_FAQ.md]: You don't actually mention anywhere that the problem is that the subnet for the VPC overlaps with `10.0.0.0/8`

**Comment by kayousterhout** [comment]
I updated this -- thanks for the comments!

**Pull request merged**